.github/workflows/python-pr-open.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Python Package
```

Comment:
Should this be called Build Python Package? It isn't doing so, is it?

Reply:
aye,  copy paste error. thanks.